_posts/2018-02-15-upgrading-a-monolith.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Upgrading a monolith from Rails 4.0 to 5.1"
```

Comment:
@ercohen14 Maybe change this to `"Upgrading a Huge Monolith from Rails 4.0 to 5.1"`